scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/JobResult.java

```diff
@@ -25,6 +25,7 @@
  */
 package org.ow2.proactive.scheduler.common.job;
 
+import java.io.IOException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)